src/gfx/shaders/Uber_frag.glsl

```diff
@@ -287,151 +398,39 @@ varying vec3 vViewPosition;
     ReflectedLight reflectedLight = ReflectedLight( vec3( 0.0 ), vec3( 0.0 ), vec3( 0.0 ));
     vec3 irradiance = ambientLightColor * PI;
 
+    float shadowMask = 1.0;
     // use loop for number
     #if NUM_DIR_LIGHTS > 1
       for (int i = 0; i < NUM_DIR_LIGHTS; i ++ ) {
```

Comment:
On more point is to think about not duplicate the code.

Reply:
Fix it
Use unroll_loop